src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionVisibilityOptions.tag

```diff
@@ -153,7 +154,7 @@
     <!-- Fix for collapsing margin problem. Reference: http://stackoverflow.com/questions/6204670 -->
     <div class="visibilityMessage overflow-hidden" id="visibilityMessage-${fqForm.questionIndex}">
         This is the visibility hint as seen by the feedback giver:
-        <ul class="text-muted background-color-warning">
+        <ul class="text-muted <%= fqForm.isQuestionHasResponses() ? "" : "background-color-warning" %>">
```

Comment:
I feel `background-color-warning` can stay for both. The yellow background helps indicate that the visibility can still be edited if the user really wants to (without it, it gives the impression that it's completely disabled).